public/video-ui/src/pages/Video/tabs/Pluto.js

```diff
@@ -28,6 +29,11 @@ export class PlutoTabPanel extends React.Component {
     return (
       <TabPanel {...rest}>
         <div className="form__group">
+          {
+            video.plutoData ?
+              <PlutoProjectLink projectId={video.plutoData.projectId}/> :
+              null
```

Comment:
this ternary can be replaced with a `&&`... 
```suggestion
            video.plutoData &&
              <PlutoProjectLink projectId={video.plutoData.projectId}/>
```

Reply:
fair enough

Reply:
personally i find it less readable that way but best to keep with the existing code style